static-metric/examples/register_integration.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*!
+
+You can integrate static metric with the `register_xxx!` macro,
+by using the `register_static_xxx!` macro provided by this crate.
+
+*/
+
+#![feature(proc_macro)]
+
+#[macro_use]
+extern crate lazy_static;
+#[macro_use]
+extern crate prometheus;
+extern crate prometheus_static_metric;
+
+use prometheus::exponential_buckets;
+use prometheus_static_metric::*;
+
+make_static_metric! {
```

Comment:
may make_static_vec_metric be better?

Reply:
Hmm, a lot of places need to be changed in order to rename (also includes a directory rename), let's do it in another PR.